data/teams.yml

```diff
@@ -75,29 +79,32 @@ teams:
+  - Nathan Denne, Development Manager, Advising London
 
 - name: "Womens Aid"
   logo: "womens-aid.png"
   graduate: true
   has_description: true
   image: "womens-aid-design.jpg"
   description:
-  - "Information about domestic abuse isn’t reaching young people or resonating with them. Women’s Aid’s consultations with young survivors revealed that they tried to deal with abusive behaviour on their own for a long time. It was only when the abuse became more severe that they felt their experiences were valid enough to qualify for help from specialist services."
-  - "On Fuse, Women’s Aid has created a new interactive online service for girls aged 14-18 to answer their questions about relationships. They then receive advice and support from experts, young survivors of abuse and their peers, to emphasise that they are not alone."
-  - "The service has been developed in consultation with over 150 young women across the country, and is currently being tested ahead of an Alpha launch."
-  - "Steph Atkins, Communications Officer at Women’s Aid: “We are indebted to CAST for their excellent support and guidance at every step of this project. The Fuse programme has given us the tools and confidence to embed user-centred design principles not only in this project but in other initiatives across the organisation. We are are proud to be collaborating with young survivors of abuse on the development of this new service, together we want to make sure that no one else has to go through what they experienced.”"
+  - Information about domestic abuse isn’t reaching young people or resonating with them. Women’s Aid’s consultations with young survivors revealed that they tried to deal with abusive behaviour on their own for a long time. It was only when the abuse became more severe that they felt their experiences were valid enough to qualify for help from specialist services.
+  - On Fuse, Women’s Aid has created a new interactive online service for girls aged 14-18 to answer their questions about relationships. They then receive advice and support from experts, young survivors of abuse and their peers, to emphasise that they are not alone.
+  - The service has been developed in consultation with over 150 young women across the country, and is currently being tested ahead of an Alpha launch.
+  - <em>“We are indebted to CAST for their excellent support and guidance at every step of this project. The Fuse programme has given us the tools and confidence to embed user-centred design principles not only in this project but in other initiatives across the organisation. We are are proud to be collaborating with young survivors of abuse on the development of this new service, together we want to make sure that no one else has to go through what they experienced.”<em>
```

Comment:
@eghale This <em> is not closed so it is causing the Clore Social image to go funny.